README.md

```diff
@@ -5,15 +5,14 @@ Introduction
 [![Backers on Open Collective](https://opencollective.com/redbaron/backers/badge.svg)](#backers) 
 [![Sponsors on Open Collective](https://opencollective.com/redbaron/sponsors/badge.svg)](#sponsors) 
 
-RedBaron is a python library and tool powerful enough to be used into IPython
-solely that intent to make the process of **writing code that modify source
-code** as easy and as simple as possible. That include writing custom
+"RedBaron is a Python library and tool powerful enough to be used in IPython. 
+It aims to make the process of writing code that modifies source code as easy and simple as possible." That includes writing custom
```

Comment:
```suggestion
"RedBaron is a Python library and tool powerful enough to be used in IPython. 
It aims to make the process of writing code that modifies source code as easy and simple as possible." That includes writing custom
```

This was trying to be as close to original as possible. At the same time it now seems that first sentence tells nothing in particular except association with IPython. Is RedBaron designed to work best with IPython?